guppylang/prelude/quantum.py

```diff
@@ -29,12 +29,12 @@ def dirty_qubit() -> qubit: ...
 
 
 @guppy.hugr_op(quantum, quantum_op("Measure"))
-def measure_return(q: qubit @ owned) -> tuple[qubit, bool]: ...
+def measure_return(q: qubit @ owned) -> tuple[qubit, bool]: ...  # type: ignore[valid-type]
```

Comment:
You could also add `valid-type` to the `mypy: disable-error-code` pragma at the start of the file.

Same in `quantum_functional.py`